addons/mass_mailing/models/mailing.py

```diff
@@ -1288,6 +1295,18 @@ def _prepare_statistics_email_values(self):
             'formatted_date': formatted_date,
         }
 
+    @staticmethod
+    def _get_missing_mailing_list_warning():
```

Comment:
as explained above, would avoid that and just be sure field is required in view when being a mailing-list based mailing.

Reply:
see discussion above :)